BedrockServer.h

```diff
@@ -468,4 +472,17 @@ class BedrockServer : public SQLiteServer {
     chrono::steady_clock::duration _postPollChooseSockets;
     chrono::steady_clock::duration _postPollPostProcess;
     chrono::steady_clock::time_point _postPollStart;
+
+    // Network thread stuff.
+    thread _networkThread;
+    condition_variable _networkCV;
+    mutex _networkMutex;
+    set<Socket*> _networkThreadQueue;
```

Comment:
sort of misleading name given that this is a set and not a queue, and we have actual `queues` in bedrock which are `BedrockCommandQueue`s or `SScheduledPriorityQueue`s

Reply:
`_networkThreadSocketActivitySet`?